sys/nginx/default.conf.template

```diff
@@ -29,6 +29,7 @@ server {
 
     if ($request_method = POST) { set $skip_cache 1; }
     if ($query_string != "") { set $skip_cache 1; }
```

Comment:
setting properly `proxy_cache_bypass` should do the trick avoiding the if

Reply:
done